client/components/elements/Pagination.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { PropTypes } from 'react';
+
+function Pagination({ documents, nextPage, prevPage, pageData }) {
+  if (pageData.pageSize > 0) {
+    return (
+      <div>
+        <ul className="pagination center-align">
+          <li className={pageData.pageNumber < 2 ? 'disabled' : 'waves-effect'}>
+            <a onClick={prevPage} href="#!">
+              <i className="material-icons">chevron_left</i>
+            </a>
+          </li>
+          <li id="paginationText">
+            Page {pageData.pageNumber} of {pageData.totalPages}
+          </li>
+          <li className={documents.length < 9 ? 'disabled' : 'waves-effect'}>
+            <a onClick={nextPage} href="#!">
```

Comment:
You can get more or less by adding a limit to your query. Default is 9 documents and I consumed the API with 9 in mind. However, you can choose your limit and/or offset by sending a request with `endpoint/?limit=${limit}&offset=${offset}`